incubator/astro/templates/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
It would be nice to have the ability to turn off rbac creation and provide a service account for astro to use.  Some users may want to manage the service account and the rbac for it outside of the helm chart. See the [goldilocks chart](https://github.com/FairwindsOps/charts/tree/master/incubator/goldilocks) for an example of what I'm talking about.

Reply:
Also, not necessarily blocking for this PR, but a nice to have for a production-ready chart.